### PR TITLE
Updated path for compiled front end code.

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Changed the path for compiled front end code from `src/main/webapp/resources/dist/` to `src/main/webapp/dist/`.  

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

**Comment by tom114** [comment]
Remind me what the reason for this was again?  Just to remove the `dist` directory from `resources` so we can more easily search for things there?

**Comment by joshsadam** [comment]
Pretty much.

File: src/main/webapp/pages/errors/oauth_error.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <title th:text="#{error.oauth.title}">_OAUTH ERROR_</title>
     <link
       rel="stylesheet"
-      href="/resources/dist/css/access_confirmation.bundle.css"
+      href="/dist/css/access_confirmation.bundle.css"
```
Comment (tom114) [inline src/main/webapp/pages/errors/oauth_error.html]: Should this be `th:href`?

Comment (joshsadam) [reply src/main/webapp/pages/errors/oauth_error.html]: Yes, and yes.  Both fixed in 5b3276911b8bf204112ff5bcb359b0e27e547c4c

**Pull request merged**